### DISCUSSION TITLE
Server Error Importing Cables via .csv

### DISCUSSION THREAD

**Discussion by ArcticLinx711** [netbox-community/netbox#18196] [Help Wanted!] [answered]
I am getting the below error when trying to import .csv file into netbox via cable upload. Wondering if anyone else is getting this same error.

Server Error
There was a problem with your request. Please contact an administrator.

The complete exception is provided below:

<class 'UnicodeDecodeError'>

'utf-8' codec can't decode byte 0xa0 in position 165: invalid start byte

Python version: 3.10.12
NetBox version: 3.6.4
Plugins: 
  netbox_topology_views: 3.8.1
If further assistance is required, please post to the [NetBox discussion forum](https://github.com/netbox-community/netbox/discussions) on GitHub.

**Comment by candlerb** [comment, marked as answer]
The error tells you exactly what the problem is: your text file is not valid UTF-8. Specifically, it contains a byte (0xA0) which is not valid as the start of a character in UTF-8.

Maybe your CSV file was created on a Windows box and uses some Windows-specific character set like [Windows-1252](https://en.wikipedia.org/wiki/Windows-1252), or on a system using [ISO-8859-1](https://en.wikipedia.org/wiki/ISO/IEC_8859-1). The character 0xA0 is a non-breaking space in both of those encodings.  But if it were encoded in UTF-8, it would appear as the two-byte sequence 0xC2 0xA0.

Either way, you need to convert the file into valid UTF-8 to fix this. Under Linux, the tool "iconv" can be used to do this. You need to specify the source encoding that you created the CSV file with, and the target encoding as UTF-8.

Or, if you created the CSV file by exporting from a spreadsheet like Excel, select the encoding UTF-8 when you save it.
